pkg/testutils/localtestcluster/local_test_cluster.go

```diff
@@ -227,12 +226,6 @@ func (ltc *LocalTestCluster) Start(t testing.TB, baseCtx *base.Config, initFacto
 		cfg.Settings,
 		nil,
 	)
-	cfg.SystemConfigProvider = systemconfigwatcher.New(
-		keys.SystemSQLCodec,
-		cfg.Clock,
-		rangeFeedFactory,
-		zonepb.DefaultZoneConfigRef(),
-	)
```

Comment:
should we be deleting this?

Reply:
I think we can delete this but I'll look into it into a follow up.